pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+simple-optimism-node-op-geth-1  | INFO [01-22|07:28:21.089] Imported new potential chain segment     number=4,066,054 hash=fe6131..8bfd42 blocks=1         txs=1         mgas=0.000 elapsed="723.105µs" mgasps=0.000   age=1w2d9h  dirty=832.58KiB
+simple-optimism-node-op-geth-1  | INFO [01-22|07:28:21.090] Chain head was updated                   number=4,066,054 hash=fe6131..8bfd42 root=74b0c0..f0c70a elapsed="83.46µs"   age=1w2d9h
+simple-optimism-node-op-geth-1  | INFO [01-22|07:28:21.094] Imported new potential chain segment     number=4,066,055 hash=8d8383..53fdcb blocks=1         txs=1         mgas=0.000 elapsed="889.945µs" mgasps=0.000   age=1w2d9h  dirty=832.58KiB
+simple-optimism-node-op-geth-1  | INFO [01-22|07:28:21.095] Chain head was updated                   number=4,066,055 hash=8d8383..53fdcb root=0929e1..088f55 elapsed="96.886µ
+```
+
+#### Verify op-node logs
+
+```bash
+docker compose logs op-node -f
+```
+You should see logs like
+
+```bash
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.454] Sync progress                            reason="processed safe block derived from L1" l2_finalized=0f7835..05175f:4061224 l2_safe=642c13..1c1e60:4068133 l2_unsafe=642c13..1c1e60:4068133 l2_time=1,673,564,334 l1_derived=f04e19..57a277:8301109
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.456] generated attributes in payload queue    txs=1  timestamp=1,673,564,336
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.461] inserted block                           hash=50fbcd..051d3f number=4,068,134 state_root=9c515d..28b2da timestamp=1,673,564,336 parent=642c13..1c1e60 prev_randao=1a282e..c4fcb4 fee_recipient=0x4200000000000000000000000000000000000011 txs=1  update_safe=true
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.461] Sync progress                            reason="processed safe block derived from L1" l2_finalized=0f7835..05175f:4061224 l2_safe=50fbcd..051d3f:4068134 l2_unsafe=50fbcd..051d3f:4068134 l2_time=1,673,564,336 l1_derived=f04e19..57a277:8301109
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.462] generated attributes in payload queue    txs=1  timestamp=1,673,564,338
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.466] inserted block                           hash=3cca62..2a6f53 number=4,068,135 state_root=f4245e..d3f772 timestamp=1,673,564,338 parent=50fbcd..051d3f prev_randao=1a282e..c4fcb4 fee_recipient=0x4200000000000000000000000000000000000011 txs=1  update_safe=true
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:13.466] Sync progress                            reason="processed safe block derived from L1" l2_finalized=0f7835..05175f:4061224 l2_safe=3cca62..2a6f53:4068135 l2_unsafe=3cca62..2a6f53:4068135 l2_time=1,673,564,338 l1_derived=f04e19..57a277:8301109
+simple-optimism-node-op-node-1  | INFO [01-22|07:50:14.332] Received signed execution payload from p2p id=15c90f..5c92ac:4472873 peer=16Uiu2HAm4hA3Jd2hPnstG3yBUvLULV9dVZgtxmD23iH3wP2ZfYHN
+```
+
+## Mainnet
```

Comment:
Should this be ###?

Reply:
I actually want to structure it like this: https://github.com/axelarnetwork/axelar-docs/pull/306#discussion_r1084906451